subprojects/platform-play/src/main/java/org/gradle/play/plugins/PlayDistributionPlugin.java

```diff
@@ -168,6 +170,22 @@ public void execute(CreateStartScripts createStartScripts) {
         }
     }
 
+    private String getMainClass(PlayDistribution distribution) {
+        PlayPlatform playPlatform = distribution.getBinary().getTargetPlatform();
+        String playVersion = playPlatform.getPlayVersion();
+        switch (PlayMajorVersion.forPlatform(playPlatform)) {
+            case PLAY_2_2_X:
+            case PLAY_2_3_X:
+            case PLAY_2_4_X:
+            case PLAY_2_5_X:
+                return "play.core.server.NettyServer";
+            case PLAY_2_6_X:
+                return "play.core.server.ProdServerStart";
```

Comment:
I think this has actually been deprecated for a few releases (2.4.x?).  Could we try using the non-deprecated version for 2.4+?

Reply:
You are right:

https://github.com/playframework/playframework/blob/2.4.x/framework/src/play-netty-server/src/main/scala/play/core/server/NettyServer.scala#L230-L233

Will use `ProdServerStart` for 2.4 and 2.5 too.